src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -289,7 +274,7 @@ private static bool CompileProject(ProjectContext context, CompilerCommandApp ar
             var compilerName = CompilerUtil.ResolveCompilerName(context);
 
             // Write RSP file
-            var rsp = Path.Combine(intermediateOutputPath, $"dotnet-compile.{context.ProjectFile.Name}.rsp");
+            var rsp = Path.Combine(intermediateOutputPath, $"dotnet-compile.rsp");
```

Comment:
Should be. The csc specific response file doesn't have the project name either. We need to make sure that the obj folder doesn't overlap for projects (which I believe they don't).

If we put the name in there, we should do it for both files.